version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.2.25"
+version in ThisBuild := "0.3.0-SNAPSHOT"
```

Comment:
If you prefer `0.2.26-SNAPSHOT` that's fine too. No real reason to bump to 0.3 tbh.